app/cafe.py

```diff
@@ -0,0 +1,21 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{self.name}, you're not vaccinated!")
+        if date.today() > visitor.get("vaccine").get("expiration_date"):
```

Comment:
```suggestion
        if date.today() > visitor.get("vaccine", {}).get("expiration_date"):
```

This will be safer. But unfortunately, Type Error may still occur if expiration_date is None
This is impossible in this task, but generally it is a common scenario